automatic/googlechromebeta/googlechromebeta.nuspec

```
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>GoogleChromeBeta</id>
```

Shall we do

```suggestion
    <id>GoogleChrome.Beta</id>
```

To match other packages?

 * [`GoogleChrome`](https://chocolatey.org/packages/GoogleChrome)
 * `GoogleChrome.Beta`
 * [`GoogleChrome.Dev`](https://chocolatey.org/packages/GoogleChrome.Dev)
 * [`GoogleChrome.Canary`](https://chocolatey.org/packages/GoogleChrome.Canary)

---

No we should not, using `.` in the only acceptable if it ends with `.install`/`.portable` or if the application is named that way.

Better way would be `googlechrome-beta`

---

But whats in the id isn't important, it will be overridden by either the folder name, or the passed `PackageName` in the `update.ps1` script

---

since those packages are already pushed, it doesn't make any sense to change the name for them.

Or, they could be deprecated, and pushed with a new name. But not important.

---

Ok. @nsleigh Sorry for providing wrong information.